### PR TITLE
Updated Elastic Search dev-Dependency

### LINKED ISSUE #114: Elastic Search Dependency is EOL
The latest Elastic Search release supported by elastic/elasticsearch-php@2.1 [1] is Elastic Search 2.4.x to which all support ended on 28.02.2018 [2].

It looks like the elastic feature set used by MassiveSearchBundle is fully supported by Elastic Search 6 and therefore elastic/elasticsearch-php should be upgraded to the 6.0 branch.

Links:
------

[1] https://github.com/elastic/elasticsearch-php/tree/master#version-matrix
[2] https://www.elastic.co/support/eol

### PR DESCRIPTION
**Pull request by pascalvb**
Fixes #114

### CODE REVIEW

**Comment by wachterjohannes** [comment]
@pascalvb thanks for this thought and PR but I think we should to support elasticsearch `^2.0` and elasticsearch `^6.0` do you think its possible?

**Comment by pascalvb** [comment]
hi @wachterjohannes, sorry for the delay. I changed the dependency to support both ```^2.0``` and ```^6.0```, this shouldn't be an issue that way around.

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
         "zendframework/zend-stdlib": "2.3.1 as 2.0.0rc5",
         "zendframework/zendsearch": "2.*@dev",
-        "elasticsearch/elasticsearch": "^2.1",
+        "elasticsearch/elasticsearch": "^2.0|^6.0",
```
Comment (wachterjohannes) [inline composer.json]: go with `^2.1|^6.0`

**Comment by wachterjohannes** [comment]
@pascalvb could you update this branch with current develop?

**Pull request merged**